### PR TITLE
temptative clang format setup

### PR DESCRIPTION
**Pull request by jfalcou**

### CODE REVIEW

File: include/eve/concept/properly_convertible.hpp [hunk 1]
```diff
@@ -19,29 +20,30 @@ namespace eve
   {
     // Note: this traits is customizable for other types we may need later on (complex,etc...)
     template<typename T, typename U>
-    struct  has_no_loss_convertion
-          : std::bool_constant<!(std::is_floating_point_v<T> && std::is_integral_v<U>)>
-    {};
+    struct has_no_loss_convertion
+        : std::bool_constant<!(std::is_floating_point_v<T> && std::is_integral_v<U>)>
+    {
+    };
```
Comment (DenisYaroshevskiy) [inline include/eve/concept/properly_convertible.hpp:26]: Can we not do the 2 line braces? There should be an option.

File: include/eve/detail/function/simd/x86/friends.hpp [hunk 2]
```diff
@@ -9,500 +9,485 @@
...
+  constexpr auto c = categorize<wide<T, N>>();
+  constexpr auto f = to_integer(cmp_flt::eq_oq);
 
-  //================================================================================================
-  template<real_value T, typename N>
-  EVE_FORCEINLINE as_logical_t<wide<T, N>> self_eq(wide<T,N> v, wide<T,N> w) noexcept
-      requires x86_abi<abi_t<T, N>>
+  if constexpr( current_api >= avx512 )
+  {
+    if constexpr( c == category::float32x16 ) return mask16 {_mm512_cmp_ps_mask(v, w, f)};
```
Comment (DenisYaroshevskiy) [inline include/eve/detail/function/simd/x86/friends.hpp:75]: I think there was a setting to line up conditions, wasn't there?

File: include/eve/concept/properly_convertible.hpp (hunk 1 above)
Comment (jfalcou) [reply include/eve/concept/properly_convertible.hpp]: Fixed

File: include/eve/detail/function/simd/x86/friends.hpp (hunk 2 above)
Comment (jfalcou) [reply include/eve/detail/function/simd/x86/friends.hpp:75]: Can't find it. The one we waited for what that if constexpr was treated as If with respect to else.

File: include/eve/arch/cpu/wide.hpp [hunk 3]
```diff
@@ -344,49 +340,56 @@ namespace eve
...
+    }
 
     //! Post-decrementation operator
-    EVE_FORCEINLINE wide operator--(int)  noexcept  { auto that(*this); operator--(); return that; }
+    EVE_FORCEINLINE wide operator--(int) noexcept
+    {
+      auto that(*this);
+           operator--();
+      return that;
+    }
```
Comment (DenisYaroshevskiy) [inline include/eve/arch/cpu/wide.hpp:372]: What happened here? Is this by design?

**Review by DenisYaroshevskiy — COMMENTED** [review]
I think ifs are less than ideal. But it's your call

**Review by DenisYaroshevskiy — APPROVED** [review]
I guess it's going to stay "ify". We will just have to deal.

File: include/eve/detail/function/simd/x86/friends.hpp (hunk 2 above)
Comment (jfalcou) [reply include/eve/detail/function/simd/x86/friends.hpp:75]: We will remove the else whenever we run over some to guarantee alignment

File: include/eve/arch/cpu/wide.hpp (hunk 3 above)
Comment (jfalcou) [reply include/eve/arch/cpu/wide.hpp:372]: Fixed

**Pull request merged**